src/actions/actions.js

```diff
@@ -635,6 +640,24 @@ JSON: { "in": [ "authenticated", { "var": "data.roles" } ] }`;
     });
   }
 
+  async function getDeletedSubmission(req) {
+    try {
+      return await router.formio.mongoose.models.submission.findOne({
```

Comment:
This will not work for submission collections. Need to use the standard way of fetching submissions.